contract_recurring_invoice_monthly_last_day/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================================
+Contract Recurring Invoice Monthly Last Day
+===========================================
+
+This module was written to extend the functionality of contracts to support to
+set a recurring type of monthly - last day.
+
+Configuration
+=============
+
+Just select the new recurring type in the contract.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/110/8.0
+
+This modules changes the next contract recurring invoice date calculus, so that
+in case the recurring type is "Monthly - Last day", next invoice date is
+calculated to be the last day of the corresponding month.
+
+Care has to be taken to set the first invoice date correctly (last month day)
```

Comment:
If we put a constraint, then user won't be able to change the next invoice date during the contract. I don't know if this is what we want... :)

Maybe set an onchange event so that next invoice date is computed when this new type is selected?